### DISCUSSION TITLE
Session Location?

### DISCUSSION THREAD

**Discussion by 1RonRock** [sienori/Tab-Session-Manager#686] [Q&A] [answered]
I recently removed my hard drive from my computer and replaced it with a larger drive. I have put the old drive in an enclosure and am able to pull files off of it. I need to know where the saved sessions are on the old drive and how to import them into the new TSM sessions. New drive has a clean install of WIN 10, and Firefox with TSM.

**Comment by sienori** [comment, marked as answer]
See: https://github.com/sienori/Tab-Session-Manager/wiki/Q&A:-Sessions-data-storage-location;-How-to-migrate-sessions-data-to-another-profile

**Reply by mcrmnz** [reply]
Hello sir! Is there any other way to restore sessions from non-working firefox, like read TSM file with 3rd computer? Tried several times, but this method doesn't seem to be working for me - in step 7, after firefox launching, the session list is blank empty as it is after TSM installation. Or maybe there are more recent methods/instructions?

UPD: It actually worked after i copied `1782160246ssensosi.files` folder as well! Thanks!

**Comment by tps125** [comment]
Where are the sessions saved in Chrome/Chromium?  I have an instance of Chromium that I am unable to start and would like to migrate the sessions to another instance.

**Reply by sienori** [reply]
In Windows Chrome, it was found in the following file:
`C:\Users\USER\AppData\Local\Google\Chrome\User Data\Default\IndexedDB\chrome-extension_apckmmkkalmmljfdlejfiaafhkkhbaeg_0.indexeddb.leveldb`
I don't know how to migrate to another profile...

**Reply by Mondriaan123** [reply]
And where are the sessions saved in firefox?

Thank you,
Mondriaan
